presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -131,6 +132,9 @@
 public class HiveMetadata
         implements ConnectorMetadata
 {
+    @VisibleForTesting
```

Comment:
Remove this and make the field public. Since we will be adding this to table properties, the value itself is very public.